src/Components/Rating/Rating.scss

```diff
@@ -0,0 +1,68 @@
+@import '../../styles/Colors';
+
+textarea {
+    resize:none;
+    width:100%;
+}
+.text-btns{
+    margin-bottom:10px;
+    width:100%;
+    .btns {
+        width:130px;
+        height:40px;
+        // border-radius: 0 !important;
+        background-color: $grey0 !important;
+    }
+    .edit{
+        border:2px solid $grey1;
+    }
+    .active{
+        box-shadow: 1px 1px 4px $teal;
+        border: 2px solid $teal;
+    }
+}
+.input-box {
+    width:100%;
+}
+// .rate{
+//     width:100%;
+// }
+button {
+    // background-color: transparent;
+    border: none;
+    outline: none;
+    cursor: pointer;
+}
+.on {
+    background-color: transparent !important;
+    color: $teal;
+}
+.off {
+    background-color: transparent !important;
+    color: $grey2;
+}
+.true{
+    background-color: $teal;
```

Comment:
Please use some different name than built in names
dont use on off true false, you are open to use active, inactive and other names which suits better